sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientBuilder.java

```diff
@@ -1212,6 +1224,17 @@ CosmosAsyncClient buildAsyncClient(boolean logStartupInfo) {
     public CosmosClient buildClient() {
         StopWatch stopwatch = new StopWatch();
         stopwatch.start();
+
+        if (Configs.shouldOptInDefaultCircuitBreakerConfig()) {
+            System.setProperty(
```

Comment:
I guess it should also work with:

```
  System.setProperty(
                "COSMOS.PARTITION_LEVEL_CIRCUIT_BREAKER_CONFIG",
                "{\"isPartitionLevelCircuitBreakerEnabled\": true}")
```

Reply:
Yeah - should be because of the `JsonSetter(nulls = Nulls.SKIP)` annotation. Fixed in next iteration.